docs/DeveloperGuide.md

```diff
@@ -290,43 +290,277 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 (For all use cases below, the **System** is the `AddressBook` and the **Actor** is the `user`, unless specified otherwise)
 
-**Use case: Delete a person**
+#### Use case: Save a student's contact
 
 **MSS**
 
-1.  User requests to list persons
-2.  AddressBook shows a list of persons
-3.  User requests to delete a specific person in the list
-4.  AddressBook deletes the person
+1. User enters command to save a student's contact.
+2. LMS parses the contact details from the user's command.
+3. LMS requests for confirmation.
+4. User confirms the details.
+5. LMS saves the contact and notifies the user.
 
-    Use case ends.
+   Use case ends.
 
 **Extensions**
 
-* 2a. The list is empty.
+* 2a. LMS detects an error in the entered data.
+  * 2a1. LMS requests for the correct data.
+  * 2a2. User enters new data.
+  * Steps 2a1-2a2 are repeated until the data entered are correct.
+  * Use case resumes from step 3.
 
-  Use case ends.
+* \*a. At any time, User chooses to cancel the saving process.
+  * \*a1. LMS requests to confirm the cancellation of the process.
+  * \*a2. User confirms the cancellation of the process.
+  * Use case ends.
 
-* 3a. The given index is invalid.
+---
 
-    * 3a1. AddressBook shows an error message.
+#### Use case: Save a student's address
 
-      Use case resumes at step 2.
+**MSS**
 
-*{More to be added}*
+1. User enters command to save a student's address.
+2. LMS parses the address details from the user's command.
+3. LMS requests for confirmation.
+4. User confirms the details.
+5. LMS saves the student's address and notifies the user.
 
-### Non-Functional Requirements
```

Comment:
I think the placeholders for the NFRs and glossary were removed

Reply:
Recovered the placeholders with new commit!